PWGHF/HFC/Tasks/taskCharmHadronsFemtoDream.cxx

```diff
@@ -195,8 +189,9 @@ struct HfTaskCharmHadronsFemtoDream {
 
   SliceCache cache;
   Preslice<aod::FDParticles> perCol = aod::femtodreamparticle::fdCollisionId;
-  Produces<o2::aod::FDResultsHF> fillFemtoResult;
-  Produces<o2::aod::FDResultsHFTrkInfo> fillFemtoResultTrkInfo;
+  Produces<o2::aod::FDHfCharm> fillFemtoResultCharm;
+  Produces<o2::aod::FDHfTrk> fillFemtoResultTrk;
+  Produces<o2::aod::FDHfColl> fillFemtoResultColl;
```

Comment:
Don't call these `fill...`. It suggests that it's a function which is not the case. These are cursors to table rows and in HF they are usually called `row...`.